modules/openy_features/openy_demo_content/modules/openy_demo_bamenities/config/install/migrate_plus.migration.block_content_amenities_block.yml

```diff
@@ -2,7 +2,8 @@ langcode: en
 status: true
 dependencies: {  }
 id: openy_demo_block_content_amenities
-migration_tags: {  }
+migration_tags:
```

Comment:
Let's split the whole pull request into two pull requests:
1. PR to @AndreyMaximov's fork with installation step changes
2. adding tags to all migrations, but take into consideration does not make sense to add the tag to migrations which are in dependencies and will be installed in any case.